docs/config/index.md

```diff
@@ -101,46 +101,74 @@ Include globs for in-source test files.
-- Importing UI libraries is expensive because they are not meant to run inside Node.js
-- Your `alias` configuration is now respected inside bundled packages
-- Code in your tests is running closer to how it's running in the browser
+- **Type:** `{ web?, ssr? }`
 
-Be aware that only packages in `deps.experimentalOptimizer?.[mode].include` option are bundled (some plugins populate this automatically, like Svelte). You can read more about available options in [Vite](https://vitejs.dev/config/dep-optimization-options.html) docs. By default, Vitest uses `experimentalOptimizer.web` for `jsdom` and `happy-dom` environments, and `experimentalOptimizer.ssr` for `node` and `edge` environments, but it is configurable by [`transformMode`](#transformmode).
+Determine the transform method for all modules inported inside a test that matches the glob pattern. By default, relies on the environment. For example, tests with JSDOM environment will process all files with `ssr: false` flag and tests with Node environment process all modules with `ssr: true`.
 
-This options also inherits your `optimizeDeps` configuration (for web Vitest will extend `optimizeDeps`, for ssr - `ssr.optimizeDeps`). If you redefine `include`/`exclude` option in `deps.experimentalOptimizer` it will extend your `optimizeDeps` when running tests. Vitest automatically removes the same options from `include`, if they are listed in `exclude`.
+#### server.transformMode.ssr
 
-::: tip
-You will not be able to edit your `node_modules` code for debugging, since the code is actually located in your `cacheDir` or `test.cache.dir` directory. If you want to debug with `console.log` statements, edit it directly or force rebundling with `deps.experimentalOptimizer?.[mode].force` option.
-:::
+- **Type:** `string[]`
+- **Default:** `[]`
+
+Use SSR transform pipeline for all modules inside specified tests.<br>
+Vite plugins will receive `ssr: true` flag when processing those files.
+
+#### server.transformMode&#46;web
+
+- **Type:** `string[]`
+- **Default:** `[]`
+
+First do a normal transform pipeline (targeting browser), then do a SSR rewrite to run the code in Node.<br>
+Vite plugins will receive `ssr: false` flag when processing those files.
+
+#### server.debug
+
+- **Type:** `{ dumpModules?, loadDumppedModules? }`
 
-#### deps.external
+Vite-Node debugger options.
+
+#### server.debug.dumpModules
+
+- **Type:** `boolean | string`
+
+Dump the transformed module to filesystem. Passing a string will dump to the specified path.
+
+#### server.debug.loadDumppedModules
+
+- **Type:** `boolean`
+
+Read dumped module from filesystem whenever exists. Useful for debugging by modifying the dump result from the filesystem.
+
+#### server.deps
+
+- **Type:** `{ external?, inline?, ... }`
+
+Handling for dependencies resolution.
+
+#### server.deps.external
 
 - **Type:** `(string | RegExp)[]`
-- **Default:** `[/\/node_modules\//]`
+- **Default:** `['**/node_modules/**']`
```

Comment:
Looks like you incorrectly merged docs here. It should be `/\/node_modules\//` and it misses the paragraph about `moduleDirectories`